### ISSUE TITLE
ModuleNotFoundError: tutorial/parse.py

### ISSUE THREAD

**Issue by dcslagel** [FRI-Energy-Analytics/alaska#36]
**Describe the bug**
When trying to run tutorial/parse.py, the script fails with `ModuleNotFoundError: No module named 'aliaser'`

**To Reproduce**
1. clone the alaska repo
2. cd alaska
3. pip install -e .
4. pip install welly
5. python tutorials/parse.py
6. output
```python
Traceback (most recent call last):
  File "tutorials/parse.py", line 18, in <module>
    from aliaser import Alias
ModuleNotFoundError: No module named 'aliaser'
```

**Expected behavior**
Print the panda data frame output from the parse.py processing

**Screenshots**
N/A

**System (please complete the following information):**
 - OS: Darwin
 - Python version Python 3.8.6 
 - AlasKA Version v0.3-beta+34.g5aa5745
 - PyTorch Version 1.7.0

**Additional context**
N/A

**Comment by jessepisel** [comment]
Nice catch! This is a remnant from a development version of the package. I will update this afternoon along with the data path for the tutorial

**Comment by dcslagel** [comment]
@jessepisel,

This issue can probably be closed now that #39  'Fix ModuleNotFoundErrors in tutorial examples' merged to main.  I verified the issues in not seen in the master branch by following the steps in 'To Reproduce' from the issue description above.  (There is a subsequent error, I will make a new issue for that).

Thank you,
DC
